README.md

```diff
@@ -142,3 +142,122 @@ Options:
 ```sh
 ts-node src/index eth:vesting --csv test.csv --from 0xd84b338b06222295a9ac1f1e81722f0c3a354884 --password 1234
 ```
+
+## Walkthrough
+
+This walkthrough will proceed through each of the core utilities of the 
+Ethereum side of the tool using `openethereum` v3.0.0. The tool was updated
+to separate the signing and the broadcasting functionality so it may work
+differently than the behavior you expected in the past.
+
+### Using `openethereum`
+
+Set up a new account in the local keystore. For this walkthrough we'll be using
+the Goerli test network.
+
+```zsh
+openethereum account new --chain goerli
+```
+
+Type your password in twice and you will be given a fresh and newly generated
+account.
+
+```zsh
+Please note that password is NOT RECOVERABLE.
+Type password: 
+Repeat password: 
+0x5b01b9990cd3d7b4ddaff97665eff702d1ccb2a2
+```
+
+Now since we will only be using openethereum for its keystore, we can start
+the service using the following flags to disable networking and syncing.
+
+```zsh
+openethereum --chain goerli --max-peers 0 --ws-apis all
+```
+
+### Using injection-tool
+
+Now open a new terminal, we'll be using most of the Ethereum functionality
+in injection-tool now to manage our own deployment of a `FrozenToken` and
+`Claims` contract.
+
+> Note: The instruction below assume you have the injection-tool source code
+> locally and that you're using `ts-node` to run the commands. An alternative
+> is to install the injection-tool from NPM with `npm i @w3f/injection-tool -g`.
+> After installing you would replace `ts-node src/index` in the commands below
+> with the `injection-tool` command.
+
+```zsh
+ts-node src/index eth:frozenToken-deploy --nonce 0 --output frozen.raw.tx --owner 0x5b01b9990cd3d7b4ddaff97665eff702d1ccb2a2 --from 0x5b01b9990cd3d7b4ddaff97665eff702d1ccb2a2 --password <your_password>
+```
```

Comment:
--owner & --from should indicate to use the address you just generated above <your_address>